src/main/java/fi/vm/yti/datamodel/api/v2/service/SchemaService.java

```diff
@@ -67,17 +67,15 @@ private void checkAndAddPropertyFeature(JsonNode node, Model model, Resource pro
 				} 
 				else if(key == "enum") {
 					if (!propertyNode.isEmpty()) {
-						RDFNode[] elements = new RDFNode[propertyNode.size()];
 						Bag bag = model.createBag();
 						
 						for(int i = 0; i < propertyNode.size(); i++){
 							if (node.get("type").asText() == "boolean")
```

Comment:
This is probably not doing what you think it is doing. 
See for example https://www.baeldung.com/java-compare-strings